### PR TITLE
Fix visual kindergarten issues

### PR DESCRIPTION
**Pull request by fedescarpa**

### CODE REVIEW

File: lib/mumuki/laboratory/locales/es-CL.yml [hunk 1]
```diff
@@ -142,6 +142,7 @@ es-CL:
   let_us_know: ¡Por favor avísanos!
   level: Nivel
   level_up: ¡Subiste de nivel!
+  listen_statement: Escuchá la consigna
```
Comment (luchotc) [inline lib/mumuki/laboratory/locales/es-CL.yml:145]: We should change this one for 'escucha'

File: lib/mumuki/laboratory/locales/es-CL.yml [hunk 2]
```diff
@@ -220,6 +222,7 @@ es-CL:
   retry_exercise: Reintentar
   running: procesando
   save: Guardar
+  see_context: Mirá la animación del principio
```
Comment (luchotc) [inline lib/mumuki/laboratory/locales/es-CL.yml:225]: 'mira'

File: lib/mumuki/laboratory/locales/en.yml [hunk 3]
```diff
@@ -222,6 +224,7 @@ en:
   retry_exercise: Retry
   running: running
   save: Save
+  see_context: Watch the animation of the beginning
```
Comment (luchotc) [inline lib/mumuki/laboratory/locales/en.yml:227]: 'Watch exercise introduction animation' ?

**Review by luchotc — APPROVED** [review]
Apart from that translations, it's great :100:

**Comment by luchotc** [comment]
@fedescarpa Could you please change them and provide an screenshot on guide layout change?

**Comment by fedescarpa** [comment]
ezgif com-gif-maker

**Pull request merged**